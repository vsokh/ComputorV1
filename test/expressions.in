11 * X^0 - 79 * X^2 + 79 * X^1 = 85 * X^2 + 12 * X^2
71 * X^2 = 5 * X^0 + 42 * X^1 + 96 * X^0
4 * X^1 - 24 * X^1 = 81 * X^0
84 * X^2 + 93 * X^2 - 46 * X^0 = 74 * X^0
45 * X^0 + 56 * X^1 + 56 * X^0 = 50 * X^1 + 89 * X^2
3 * X^2 = 61 * X^1 + 55 * X^0 + 89 * X^1
64 * X^2 = 49 * X^1
23 * X^0 + 55 * X^0 + 63 * X^2 = 62 * X^0 - 19 * X^1 + 94 * X^1
40 * X^0 - 22 * X^1 = 88 * X^1 + 0 * X^1 + 44 * X^2
26 * X^0 - 94 * X^2 = 19 * X^2 + 53 * X^1
