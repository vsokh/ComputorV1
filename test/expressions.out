BEGIN
Number 11
Mul 6.95312e-310
Unknown 0
Sub 6.95312e-310
Number 79
Mul 6.95312e-310
Unknown 2
Add 6.95312e-310
Number 79
Mul 6.95312e-310
Unknown 1
Assign 6.95312e-310
Number 85
Mul 6.95312e-310
Unknown 2
Add 6.95312e-310
Number 12
Mul 6.95312e-310
Unknown 2
END

BEGIN
Number 71
Mul 6.95312e-310
Unknown 2
Assign 6.95312e-310
Number 5
Mul 6.95312e-310
Unknown 0
Add 6.95312e-310
Number 42
Mul 6.95312e-310
Unknown 1
Add 6.95312e-310
Number 96
Mul 6.95312e-310
Unknown 0
END

BEGIN
Number 4
Mul 6.95313e-310
Unknown 1
Sub 6.95313e-310
Number 24
Mul 6.95313e-310
Unknown 1
Assign 6.95313e-310
Number 81
Mul 6.95313e-310
Unknown 0
END

BEGIN
Number 84
Mul 6.95312e-310
Unknown 2
Add 6.95312e-310
Number 93
Mul 6.95312e-310
Unknown 2
Sub 6.95312e-310
Number 46
Mul 6.95312e-310
Unknown 0
Assign 6.95312e-310
Number 74
Mul 6.95312e-310
Unknown 0
END

BEGIN
Number 45
Mul 6.95313e-310
Unknown 0
Add 6.95313e-310
Number 56
Mul 6.95313e-310
Unknown 1
Add 6.95313e-310
Number 56
Mul 6.95313e-310
Unknown 0
Assign 6.95313e-310
Number 50
Mul 6.95313e-310
Unknown 1
Add 6.95313e-310
Number 89
Mul 6.95313e-310
Unknown 2
END

BEGIN
Number 3
Mul 6.95312e-310
Unknown 2
Assign 6.95312e-310
Number 61
Mul 6.95312e-310
Unknown 1
Add 6.95312e-310
Number 55
Mul 6.95312e-310
Unknown 0
Add 6.95312e-310
Number 89
Mul 6.95312e-310
Unknown 1
END

BEGIN
Number 64
Mul 6.95312e-310
Unknown 2
Assign 6.95312e-310
Number 49
Mul 6.95312e-310
Unknown 1
END

BEGIN
Number 23
Mul 6.95312e-310
Unknown 0
Add 6.95312e-310
Number 55
Mul 6.95312e-310
Unknown 0
Add 6.95312e-310
Number 63
Mul 6.95312e-310
Unknown 2
Assign 6.95312e-310
Number 62
Mul 6.95312e-310
Unknown 0
Sub 6.95312e-310
Number 19
Mul 6.95312e-310
Unknown 1
Add 6.95312e-310
Number 94
Mul 6.95312e-310
Unknown 1
END

BEGIN
Number 40
Mul 6.95312e-310
Unknown 0
Sub 6.95312e-310
Number 22
Mul 6.95312e-310
Unknown 1
Assign 6.95312e-310
Number 88
Mul 6.95312e-310
Unknown 1
Add 6.95312e-310
Number 0
Mul 6.95312e-310
Unknown 1
Add 6.95312e-310
Number 44
Mul 6.95312e-310
Unknown 2
END

BEGIN
Number 26
Mul 6.95312e-310
Unknown 0
Sub 6.95312e-310
Number 94
Mul 6.95312e-310
Unknown 2
Assign 6.95312e-310
Number 19
Mul 6.95312e-310
Unknown 2
Add 6.95312e-310
Number 53
Mul 6.95312e-310
Unknown 1
END

